package.json

```diff
@@ -18,6 +18,18 @@
       "title": "Works for Private Browsing",
       "type": "bool",
       "value": true
+    },
+    {
+      "name": "maxOpenInNextWindow",
+      "title": "Open in next available window",
```

Comment:
@robsonsobral I get your point :smile: Well, this feature wouldn't even be needed if the point is to never "exceed" the max number of tabs :stuck_out_tongue: